app/models/sign_in/session_container.rb

```diff
@@ -41,5 +41,18 @@ def initialize(session:, # rubocop:disable Metrics/ParameterLists
     def persisted?
       false
     end
+
+    def context
+      access_token.to_s.merge(context_details)
```

Comment:
If we're going to organize the context this way I'd rather we explicitly specify the fields in the context.